api/tests/unit/infrastructure/utils/string-utils_test.js

```diff
@@ -72,6 +72,10 @@ describe('Unit | Utils | string-utils', () => {
 
   describe('#normalizeAndSortChars', () => {
 
+    it('test', () => {
+      expect(normalizeAndSortChars(('12345'))).to.equal('12345');
+    });
+
     it('should normalize and sort chars of the string "ABCDEFGHI"', () => {
       expect(normalizeAndSortChars(('ABCDEFGHI'))).to.equal('ABCDEFGHI');
```

Comment:
Question: Pour que le test soit vraiment pertinent, il  ne faudrait pas que l'entrée soit différente de la sortie ? genre : "ACBDGHEFI"

Reply:
Il y a un test plus "pertinent" juste en dessous avec des caractères spéciaux. Celui ci est surtout là pour montrer que l'on ne modifie pas la chaîne en paramètre s'il n'y a rien à faire.

Reply:
Le test du dessous test spécifiquement les caractères spéciaux (si on se fit à son IT). Mais là juste en me fiant à l'intitulé du test, je m'attends à ce que ça trie et normalise, mais l'entrée est déjà en majuscule et trier correctement. 

Ceci dit du coup, le test du dessous et peut être suffisant et celui ci en trop non ?

Reply:
J'ai modifié le test le plus complet afin d'inclure des chiffres et supprimé les deux tests précédents.